docs/core-concepts/libraries/libraries-with-cli.md

```diff
@@ -331,186 +329,261 @@ using System.Threading.Tasks;
 #endif
 ```
 
-And that's it! Because `PORTABLE328` is now recognized by the compiler, the PCL Profile 328 library generated by a compiler will not include `System.Net.Http` or `System.Threading.Tasks`.
+Because `PORTABLE328` is now recognized by the compiler, the PCL Profile 328 library generated by a compiler will not include `System.Net.Http` or `System.Threading.Tasks`.
 
-## How to use multiple projects
+Now you can build.
+
+```
+$ dotnet restore
+$ dotnet build
+```
 
-A common need for larger libraries is to place functionality in different projects.  For example, a library supplying functionality to both C# and F# consumers may have a core project, a C# project, and an F# project.  Here's how a potential folder structure would look:
+Your `/bin/Debug` folder will look like this:
 
 ```
-/MyLibrary
-|__global.json
-|__/MyLibrary.Core
-   |__Sources
-   |__project.json
-|__/MyLibrary.CSharp
-   |__Sources
-   |__project.json
-|__/MyLibrary.FSharp
-   |__Sources
-   |__project.json
+$ ls bin/Debug
+
+portable-net40+sl50+netcore45+wpa81+wp8/
+netstandard1.5/
 ```
 
-The `global.json` file would look like this:
+## How to use multiple projects
+
+A common need for larger libraries is to place functionality in different projects.
+
+Imagine you wished to build a library which could be consumed in idiomatic C# and F#.  A common approach to this is to factor all of the logic into a centralized place, and have a C# and F# project which call into that central place.
 
-```javascript
+### Project-to-project referencing
+
+To reference a project, you need to do two things:
+
+1. Understand the name and version number of the project you wish to reference.
+2. List that project as a dependeny using the name and version number from (1).
+
+In the above case, you may wish to set up the `project.json` for **AwesomeLibrary.Core** as follows:
```

Comment:
In the above case -> In the previous example
Also, you might need to introduce this library name earlier when you're setting the scene with the "Imagine..." sentence.